packages/web/examples/calculator/src/components/Loader/index.jsx

```diff
@@ -0,0 +1,14 @@
+import "./styles.css";
```

Comment:
ReactComponent is of ReactElement or ReactNode has to return something;
Consider ReactNode is returning null.